pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -257,6 +257,8 @@ func (am *Alertmanager) SaveAndApplyDefaultConfig(ctx context.Context) error {
 		Default:                   true,
 		ConfigurationVersion:      fmt.Sprintf("v%d", ngmodels.AlertConfigurationVersion),
 		OrgID:                     am.orgID,
+		// Default configuration should always be valid.
+		IsValid: true,
```

Comment:
What happens if it's not, though? Do we validate it on startup?

Reply:
I was really reluctant on adding this field. I'll surely give it a second look before marking this PR as a non-draft.